presto-prometheus/src/test/java/io/prestosql/plugin/prometheus/TestPrometheusRecordSet.java

```diff
@@ -51,7 +52,7 @@
     private URI dataUri;
     private static final Metadata METADATA = createTestMetadataManager();
     public static final TypeManager TYPE_MANAGER = new InternalTypeManager(METADATA);
-    static final Type varcharMapType = TYPE_MANAGER.getType(TypeSignature.parseTypeSignature("map(varchar, varchar)"));
+    static final Type varcharMapType = TYPE_MANAGER.getType(mapType(VARCHAR.getTypeSignature(), VARCHAR.getTypeSignature()));
```

Comment:
Why not get this from MetadataUtil like in `TestPrometheusMetadata.java`?

Reply:
I just started with this and then did the metadatautil changes after, so I may have missed a few I think. I'll double check.